### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by Mythicbeanz** [jacobfortune/Songs-Of-War-Addon#6] [Q&A] [answered]
Hey your discord links are expired can you make some new ones?

**Comment by lesleyespire** [comment, marked as answer]
<https://discord.gg/tgYhJryaCM>
discord.gg<https://discord.gg/tgYhJryaCM>
[X]<https://discord.gg/tgYhJryaCM>

Sent from my iPhone

On 15 Feb 2024, at 8:05 am, Jacob F. ***@***.***> wrote:

﻿ Sorry about that
Sent from my iPhone

On 14 Feb 2024, at 12:54 am, Mythicbeanz ***@***.***> wrote:

﻿

Hey your discord links are expired can you make some new ones?
